src/org/jetbrains/plugins/scala/project/Versions.scala

```diff
@@ -23,13 +24,18 @@ object Versions  {
     case Dotty => loadVersionsOf(Entity.Dotty)
   }
 
+  def loadHydraVersions(): Array[String] = loadVersionsOf(Entity.Hydra)
```

Comment:
Remove the empty parens (this is for consistency w/ the below `loadSbtVersions` declaration)